code/modules/antagonists/heretic/items/madness_mask.dm

```diff
@@ -68,7 +68,7 @@
 
 		if(human_in_range.getStaminaLoss() >= 85 && DT_PROB(30, delta_time))
 			human_in_range.emote(pick("giggle", "laugh"))
-			human_in_range.adjustStaminaLoss(10)
+			human_in_range.adjustStaminaLoss(15) //SKYRAT EDIT - Original: 10
```

Comment:
this can't afford to be buffed anymore, due to the fact stamina regens extremely slowly now, as compared to during the first PR.

Reply:
Sure. I'll leave it at the baseline amount for the moment.
I thought that the general HP buff would have effected it, but I'll see how it turns out if people use it.